libs/blocks/chart/chart.js

```diff
@@ -330,18 +333,17 @@ export const pieSeriesOptions = (size) => {
 /**
  * Returns object of echart options
  * @param {string} chartType
- * @param {object} data
+ * @param {object} dataset
+ * @param {object} series
+ * @param {array} headers
  * @param {array} colors
  * @param {string} size
+ * @param {array} units
  * @returns {object}
  */
-export const getChartOptions = (chartType, data, colors, size, chart) => {
-  const headers = data?.data?.[0];
+export const getChartOptions = (chartType, dataset, series, headers, colors, size, units = []) => {
```

Comment:
don't have to change but this would be a good candidate for object param instead